client/packages/purchase_orders/src/ListView/ListView.tsx

```diff
@@ -45,12 +47,18 @@ const ListView: FC = () => {
   };
 
   const navigate = useNavigate();
-  // const modalController = useToggle();
   const {
     query: { data, isError, isLoading },
   } = usePurchaseOrderList(listParams);
   const pagination = { page, first, offset };
 
+  useEffect(() => {
+    const disabledRows = (data?.nodes ?? [])
+      .filter(row => row.status !== PurchaseOrderNodeStatus.New)
+      .map(({ id }) => id);
+    setDisabledRows(disabledRows);
+  }, [data, setDisabledRows]);
+
```

Comment:
I was just wondering if users would not be able to edit anything once a PO is not new?
I can imagine once confirmed, things like shipping method, expected delivery time, etc might still be TBC or might change

Reply:
Yeah, I'm not sure about that precisely. I'll make a TO-DO note to clarify what statuses can be edited/deleted. (Pretty sure only "New" should be deleted, but not sure about editing)